spec.html

```diff
@@ -26701,12 +26701,16 @@
           1. Let _truncatedStringFiller_ be a new String value consisting of repeated concatenations of _filler_ truncated to length _fillLen_.
           1. Return a new String value computed by the concatenation of _S_ and _truncatedStringFiller_.
         </emu-alg>
-        <emu-note>The first argument _maxLength_ will be clamped such that it can be no smaller than the length of the *this* value.</emu-note>
-        <emu-note>The optional second argument _fillString_ defaults to *" "* (a String consisting of 0x0020 SPACE).</emu-note>
+        <emu-note>
+          <p>The first argument _maxLength_ will be clamped such that it can be no smaller than the length of the *this* value.</p>
```

Comment:
gotcha. @bterlson maybe we could make ecmarkup add these automatically, to remove the boilerplate?